instrumentation/google.golang.org/grpc/otelgrpc/stats_handler.go

```diff
@@ -143,8 +143,11 @@ func (c *config) handleRPC(ctx context.Context, rs stats.RPCStats, isServer bool
 	span := trace.SpanFromContext(ctx)
 	gctx, _ := ctx.Value(gRPCContextKey{}).(*gRPCContext)
 	var messageId int64
-	metricAttrs := make([]attribute.KeyValue, 0, len(gctx.metricAttrs)+1)
-	metricAttrs = append(metricAttrs, gctx.metricAttrs...)
+	var metricAttrs []attribute.KeyValue
+	if gctx != nil {
+		metricAttrs := make([]attribute.KeyValue, 0, len(gctx.metricAttrs)+1)
```

Comment:
```suggestion
		metricAttrs = make([]attribute.KeyValue, 0, len(gctx.metricAttrs)+1)
```

It doesn't need to be redefined, does it?

Reply:
ah, thanks. My wrong.